src/frontend/components/UI/Sidebar/components/CurrentDownload/index.scss

```diff
@@ -0,0 +1,40 @@
+.currentDownload {
+  font-size: var(--text-sm);
+  padding: var(--space-md) 0;
+  transition: 300ms;
+  margin-top: var(--space-sm);
+
+  & > .gameTitle {
+    font-size: var(--text-md);
+    transition: 600ms;
+    font-weight: var(--bold);
+  }
+
+  &:hover > .gameTitle {
+    color: var(--accent-overlay);
+    transition: 600ms;
+  }
+
+  .downloadStatus {
+    color: var(--success);
+    font-family: var(--secondary-font-family);
+    font-style: italic;
+    font-weight: normal;
+    font-size: var(--text-sm);
+    line-height: var(--text-sm);
+  }
+
+  .MuiLinearProgress-bar1Determinate {
+    background-color: var(--success);
+  }
+
+  .statusIcon {
+    display: none;
+    margin-inline-end: 10px;
```

Comment:
this is the important change in this file, the rest is just that I moved the file to be a `scss` file